python/lsst/meas/extensions/scarlet/scarletDeblendTask.py

```diff
@@ -91,7 +91,7 @@ def _checkBlendConvergence(blend, f_rel):
 def _getPsfFwhm(psf):
     """Calculate the FWHM of the `psf`
     """
-    return psf.computeShape().getDeterminantRadius() * 2.35
+    return psf.computeShape(psf.getAveragePosition()).getDeterminantRadius() * 2.35
```

Comment:
This method doesn't seem to be used anywhere. If it's just not showing up in my Github search, then this should take position as an argument and compute the PSF shape at that position.

Reply:
I think that whatever called it was removed some time ago, so you should be able to just remove `_getPsfFwhm`.

Reply:
Ah great, note that the use here (and above!) were found by searching through the code, and did _not_ show up in the logs in RC2 processing.  So I think this really is unused.

Reply:
Upon further review/consideration, I think this should be removed on DM-34777, and this PR should be closed as a noop.